wom/src/main/scala/wom/types/WomArrayType.scala

```diff
@@ -41,11 +41,6 @@ sealed trait WomArrayType extends WomType {
     case womArray: WomArray if (allowEmpty || womArray.nonEmpty) && memberType.isCoerceableFrom(womArray.womType.memberType) =>
       womArray.map(v => memberType.coerceRawValue(v).get) // .get because isCoerceableFrom should make it safe
     case WomArrayLike(womArray) if this.isCoerceableFrom(womArray.womType) => coercion.apply(womArray)
-    case womValue: WomValue if memberType.isCoerceableFrom(womValue.womType) =>
```

Comment:
How is this match still exhaustive with this case removed? The `WomValue` type presumably still exists, so what happens when you get one?

Reply:
It isn't, but that's ok because it's a `PartialFunction[Any, WomValue]`.

In fact it never was complete - the `case womValue: WomValue` wasn't a catch-all because it still had a `if memberType.isCoerceableFrom(womValue.womType)` clause